### ISSUE TITLE
fix tests

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#76]
Adding auth has broken all tests. 

Add a setup file to import some basic mock data that can be imported in any test file.

**Comment by JMurphyWeb** [comment]
This is now done and should be closed once #84 is merged in

**Comment by JMurphyWeb** [comment]
fixed and has been merged into master.
